pkgs/tools/package-management/cargo-download/default.nix

```diff
@@ -1,14 +1,19 @@
 { stdenv, lib, fetchgit, darwin, buildPlatform
 , buildRustCrate, buildRustCrateHelpers, defaultCrateOverrides }:
 
-((import ./Cargo.nix {
+(((import ./Cargo.nix {
   inherit lib buildPlatform buildRustCrate buildRustCrateHelpers fetchgit;
   cratesIO = import ./crates-io.nix { inherit lib buildRustCrate buildRustCrateHelpers; };
 }).cargo_download {}).override {
   crateOverrides = defaultCrateOverrides // {
     cargo-download = attrs: {
       buildInputs = lib.optional stdenv.isDarwin
         darwin.apple_sdk.frameworks.Security;
+      broken = true;
```

Comment:
```suggestion
```

Reply:
we'll just pretend that didn't happen :dancers: